### DISCUSSION TITLE
Migrate from Restic Uploader to Kopia

### DISCUSSION THREAD

**Discussion by Nabsku** [vmware-tanzu/velero#7739] [Community support Q&A] [answered]
Hello, just wondering, can I switch from Restic to Kopia painlessly if I've already done backups or do I need to do something to facitiliate a migration to Kopia?

**Comment by sseago** [comment, marked as answer]
If you change the uploader type to kopia, then all new backups will use kopia instead of restic. Regardless of what uploader type is set to for new backups, both restic and kopia existing backups can be restored. Eventually restic will be removed entirely, but it will be several releases after it is deprecated, so there will be plenty of time to make sure that all of your workloads are backed up with kopia before that happens.

**Reply by Nabsku** [reply]
Much appreciated. That was my understanding of it, just wanted to get clarification on this. Thanks!
